src/core/service.c

```diff
@@ -3047,6 +3090,15 @@ static void service_notify_message(Unit *u, pid_t pid, char **tags, FDSet *fds)
                 service_add_fd_store_set(s, fds, name);
         }
 
+        e = strv_find_startswith(tags, "WDOGSEC=");
```

Comment:
please just call this WATCHDOG_USEC= the same as the env var. There's no need to abbreviate this, and it certainly doesn't help readability.